docs/index.pug

```diff
@@ -0,0 +1,11 @@
+extends /layout/base.pug
```

Comment:
> Any problems with that?

No. We can use MD file as the main page. But we need to prepare a separate [template](https://github.com/cksource/docs-builder/tree/t/1/template) for the page. But, it isn't a problem.